bool apriltagStatus

int64[] apriltagID

float64[] apriltagCenter_x
float64[] apriltagCenter_y

float64[] apriltagHomography_00
float64[] apriltagHomography_01
float64[] apriltagHomography_02
float64[] apriltagHomography_10
float64[] apriltagHomography_11
float64[] apriltagHomography_12
float64[] apriltagHomography_20
float64[] apriltagHomography_21
float64[] apriltagHomography_22

float64[] apriltagCorner_x1
float64[] apriltagCorner_y1
float64[] apriltagCorner_x2
float64[] apriltagCorner_y2
float64[] apriltagCorner_x3
float64[] apriltagCorner_y3
float64[] apriltagCorner_x4
float64[] apriltagCorner_y4

float64[] apriltagDistance
